main.cpp

```diff
@@ -112,26 +122,40 @@ struct Car
     int moveForward(int distanceTraveled, int time);
     bool stopMovement(bool applyBrakes);
     bool turn(bool rotateSteeringWheel);
-    
+    void honk();
 };
 
-int Car::moveForward(int distanceTraveled, int time)
+Car::Car()
 {
+    std::cout << "Car being constructed!" << std::endl; 
+}
+
+int Car::moveForward(int distanceTraveled = 60, int time = 4)
+{
+    std::cout << distanceTraveled / time << std::endl;
     return distanceTraveled / time;
 }
 
-bool Car::stopMovement(bool applyBrakes)
+bool Car::stopMovement(bool applyBrakes = false)
 {
     return applyBrakes ? true : false;
 }
 
-bool Car::turn(bool rotateSteeringWheel)
+bool Car::turn(bool rotateSteeringWheel = true)
 {
     return rotateSteeringWheel ? true : false; 
 }
```

Comment:
Default arguments should be defined where the function is declared, not where the function is defined. 
- [ ] revise